docs/src/main/asciidoc/javascript/config.js

```diff
@@ -13,7 +13,7 @@ if(tables){
         if (table.classList.contains('searchable')) { // activate search engine only when needed
```

Comment:
Aren't `javascript/config.js` and `stylesheet/config.js` in this repo too?

BTW, I'm strongly against reformatting all the file. Or do it in a separate commit then add a commit with your changes.

Reply:
I don't see any reformat changes. It's in the other PR, no?

Reply:
It's in the other PR but shouldn't be. The master config.js is in this repo.

Reply:
Well the other PR contains the copy of docs from this repo, like for all guides.

Reply:
Should I change something here?